packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.service.js

```diff
@@ -18,10 +19,18 @@ import {
 
 export default class FlavorsList {
   /* @ngInject */
-  constructor($q, CucPriceHelper, OvhApiCloudProjectFlavor) {
+  constructor(
+    $q,
+    CucPriceHelper,
```

Comment:
ALphabetical order please :)

Reply:
I think it is in alphabetical order already :)

Reply:
I guess I was tired